README.md

```diff
@@ -1,12 +1,12 @@
-# Bungalow Take Home Project for Data Engineer Role (V2. 2021-11-02)
+# Bungalow Take Home Project for Analytics Engineer Role (V1. 2022-03-03)
```

Comment:
Maybe?
```suggestion
# Bungalow Take Home Project for Data Analytics Engineer Role (V1. 2022-03-03)
```

Reply:
Analytics Engineering is a thing: https://www.getdbt.com/what-is-analytics-engineering/

Reply:
Never heard of it :)